packages/babel-parser/ast/spec.md

```diff
@@ -1339,7 +1340,7 @@ An attribute specified on the ImportDeclaration.
 interface ExportNamedDeclaration <: ModuleDeclaration {
   type: "ExportNamedDeclaration";
   declaration: Declaration | null;
-  specifiers: [ ExportSpecifier ];
+  specifiers: [ ExportSpecifier | ExportNamespaceSpecifier ];
```

Comment:
Can you add a note that if an `ExportNamedDeclaration` contains `ExportNamespaceSpecifier`, it must only have one `ExportNamespaceSpecifier`?